xml/System.Net.Http/HttpMessageInvoker.xml

```diff
@@ -230,11 +230,30 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="request">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="request">The HTTP request message to send.</param>
+        <param name="cancellationToken">The cancellation token to cancel operation.</param>
+        <summary>Send an HTTP request as a synchronous operation.</summary>
+        <returns>HTTP response message.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This operation blocks until the entire response including content is read.
+  
+ Most applications that are connecting to a web site will use one of the <xref:System.Net.Http.HttpClient.Send%2A> or <xref:System.Net.Http.HttpClient.SendAsync%2A> methods on the <xref:System.Net.Http.HttpClient> class.  
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">The <paramref name="request" /> was <see langword="null" />.</exception>
+        <exception cref="T:System.NotSupportedException">For HTTP/2 and higher or when requesting version upgrade is enabled by <see cref="T:System.Net.Http.HttpVersionPolicy.RequestVersionOrHigher" />.
```

Comment:
It seems one of the build warnings is showing up because this was prefixed with `T:`, but in the other usage instances it was an `F:`:

```suggestion
        <exception cref="T:System.NotSupportedException">For HTTP/2 and higher or when requesting version upgrade is enabled by <see cref="F:System.Net.Http.HttpVersionPolicy.RequestVersionOrHigher" />.
```

Also, since this enum [was introduced about a month ago](https://github.com/dotnet/runtime/issues/987), we have not updated the ref assemblies in MS Docs that contains it, so you will continue getting these warnings until we do.